core/arch/arm/kernel/sub.mk

```diff
@@ -37,7 +37,6 @@ ifeq ($(CFG_WITH_VFP),y)
 srcs-$(CFG_ARM32_core) += vfp_a32.S
```

Comment:
For commit "core: kernel: move trace_ext.c to core/kernel":

`Reviewed-by: Jerome Forissier <jerome.forissier@linaro.org>`